packages/ppx/src/declarations_to_emotion.re

```diff
@@ -4006,6 +4110,8 @@ let parse_declarations = (~loc: Location.t, property, value) => {
   | Error(_) =>
     switch (render_to_expr(~loc, property, value)) {
     | Ok(value) => Ok(value)
+    | exception (InvalidValue(v)) =>
+      Error(`Invalid_value(value ++ ". " ++ v))
```

Comment:
Why the `.` is necessary?

Reply:
and I don't see the `\`Invalid_value` error handled anywhere

Reply:
The `.` separates the property value from the error message. Which is rendered like this
```
Error: Property 'background-color' has an invalid value: 'color-mix(srgb,
       white 0%, red 0%). Both percentages can not be 0!'
```

Reply:
hmm! The error message appears inside quotes.

Reply:
We're calling Invalid_value [here](https://github.com/lubegasimon/styled-ppx/blob/467f391a39743d17f056d1b4a86045b579e99013/packages/ppx/src/declarations_to_emotion.re#L1018).